### DISCUSSION TITLE
Add Protocol to make importing new Backgrounds easier than it already is.

### DISCUSSION THREAD

**Discussion by HoppersPS4** [rocksdanister/lively#591] [Ideas]
**General:**
You know how there are certain protocol commands in Web Browsers such as **chrome://**,**file://** and a few more?
maybe lively could also get one.

And the idea would be the following:
lively:// would be the protocol name and with a few Parameters such as; 
lively://**import=/DIRECTORY/TO/VIDEO.mp4**
lively://**import=/DIRECTORY/TO/VIDEO.mp4?music=false**
lively://**importURL=https://example.com/video.mp4**
lively://**importURL=https://example.com/video.mp4?music=false**

Personally speaking as a user who has Lively running 24/7, i think this would make the program way more easier when it comes to importing new Backgrounds.
Don't get me wrong, the import system that lively features right now is great and very easy to use but i think this small idea could be very nice for lively 😀

**Comment by rocksdanister** [comment]
Media files are auto imported with `setwp` command if not already present in library:
https://github.com/rocksdanister/lively/wiki/Command-Line-Controls#set-wallpaper

Webpage and other types are not allowed due to security reasons.

**Reply by HoppersPS4** [reply]
Makes sense, thanks for the Response!
